index.js

```diff
@@ -1,6 +1,8 @@
 var urlMapper = require('url-mapper');
-var addressbar = require('addressbar');
 var pathToRegexp = require('path-to-regexp');
+try {
+  var addressbar = require('addressbar');
+} catch(e) {}
```

Comment:
`addressbar` import fails because of running in node env where window is absent.